homeworks/becseipeter/3_conditionals_loops/rock_paper_scissors.py

```diff
@@ -0,0 +1,43 @@
+all_round = int(input("How many round do you wanna play?\n"))
+player1_score = 0
+player2_score = 0
+
+while all_round % 2 ==0 :
+    all_round = int(input("Wrong number, give me a new number\n"))
+
+
+round = 1
+while round <= all_round:
+    print(f"Round: {round}")
+
+    player1 = input("Player1 select Rock, Paper, or Scissor:\n")
+    while player1 not in ["rock", "paper", "scissor"]:
+        player1 = input("Please use lower case\n")
```

Comment:
ez a hiba üzenet (és ez igaz a 20. sorra is) nem mindig ad helyes utasítást arra hogy mit csináljon máshogy a felhasználó.
Ha azt adom meg hogy "Scissor", vagy "Rock", akkor jogos hogy írjam kisbetűvel, de akkor is ezt írja ki ha azt adom meg hogy "kiscica". A hibaüzeneted valami ilyesmi lehetne: Wrong input, please provide either "rock", "paper" or "scissor".
Esetleg még azt is megcsinálhatod hogy meghívod a lower() method-ot a player1-re és a player2-re amikor megnézed hogy szerepelnek e a listában.